README.md

```diff
@@ -2,64 +2,58 @@
 
 Template to start from before starting frontend projects.
 
-## Install template
-
-- Create empty repository for your project. [learn how](#) _need a link here_
-- Clone boilerplate into project directory and navigate there.
-```bash
-$ git clone https://github.com/mate-academy/gulp-template.git project-name
-$ cd project-name
-```
-- Change remote repository to recently created
-```bash
-$ git remote set-url origin github-link-of-project
-```
-- Install dependencies
-```bash
-$ npm install
-```
-
-All packages will be installed after executing the script above.
+## Task
 
-## Development mode 
-
-```bash
-$ npm run start
-```
+You need to create function that will be generate calendar for given pair 
+(year, month).
+Calendar must be table with `thead` and `tbody`, where day it's `TD`.
 
-## Build the project
+Function `calendarTable(year, month)`
 
-```bash
-$ npm run build
-```
+## Workflow
 
-## Linters
+- Fork the repository with task
+- Clone forked repository 
+    ```bash
+    git clone git@github.com:<user_name>>/<task_repository>.git
+    ```
+- Create separate branch for the solution. Name of the branch should be the same 
+as name of your channel in slack. 
+    ```bash
+    git checkout -b fs_on_apr19_sasha
```

Comment:
btw, why do we ask this? Can they change their master and create pull request into our master?

Reply:
yes they can